tests/reg_tests/test_MPhysGeo.py

```diff
@@ -34,6 +34,34 @@
     ocsmInstalled = False
 
 
+def assert_check_totals(totals, atol, rtol):
```

Comment:
This change seems unrelated to this PR

Reply:
Right, not sure how this got in here, how can I remove this commit without undoing the changes?

Reply:
nvm, this was actually code that should've been deleted